tests/test_short_client_token.py

```diff
@@ -176,7 +176,7 @@ def test_short_client_token_lookup_delegated_patron_identifier_failure(self):
 
         # The patron identifier must not be blank.
         assert_raises_regexp(
-            ValueError, 'Token library|1234| has empty patron identifier',
+            ValueError, 'Token library\|1234\| has empty patron identifier',
```

Comment:
The pipe characters were being used when this string was compiling into a regular expression, causing some tests to pass when they shouldn't have.